src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BFHelper.java

```diff
@@ -76,11 +76,31 @@ public static void populateDynamicListBfActions(CaseData caseData) {
             bfActionType.setAction(dynamicFixedListType);
             bfActionTypeItem.setId(UUID.randomUUID().toString());
             bfActionTypeItem.setValue(bfActionType);
-
             bfActionTypeItemListAux = new ArrayList<>(Collections.singletonList(bfActionTypeItem));
-
         }
 
         caseData.setBfActions(bfActionTypeItemListAux);
     }
+
+    public static void updateWaTaskCreationTrackerOfBfActionItems(CaseData caseData) {
+        String yesterday = UtilHelper.formatCurrentDate2(LocalDate.now().minusDays(1));
+        if (caseData.getBfActions() == null || caseData.getBfActions().isEmpty()) {
```

Comment:
Minor - would also need the import added
```suggestion
        if (CollectionUtils.isEmpty(caseData.getBfActions())) {
```